### DISCUSSION TITLE
help on the "tiered caching" scripts?

### DISCUSSION THREAD

**Discussion by markkundinger** [trapexit/mergerfs#982] [Q&A]
Hi, I'm a noob.  I wanted to set up the psuedo tiered caching options introduced in the readme, but have several pieces of igorance preventing me from doing so.

1) Where inside the script do I actually enter my real drive and folder names?  The echo line in the beginning? 
` echo "usage: $0 <cache-drive> <backing-pool> <days-old>" `
lower down?
` CACHE="${1}"
BACKING="${2}"
N=${3}`

somewhere else?

2) What kind of file should I save the script as?  a .sh?

3) What command line is the best way to implement this in a crontab?

thanks!

**Comment by trapexit** [comment]
What is the use case that you feel you need this behavior?

1) I don't really understand what you're asking. As the usage says the first argument is the cache drive, the second it the backing pool. 
2) When you say "a noob" you mean noob to linux in general? Linux doesn't have extensions. The "extension" is totally irrelevant.
3) Best? You simply run it on whatever schedule you wish.
